### DISCUSSION TITLE
How to remove multiple station locations for eQSL

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#387] [Ideas]
Now we need to create multiple station locations to ensure synchronization with eQSL for occasional (weekend) trips to the country house.
It would be great if it were possible to create a station location and have a table of linked eQSL profiles with their time periods

**Comment by HB9HIL** [comment]
Hello Michael

I remember this discussion https://github.com/magicbug/Cloudlog/discussions/2762 

The main issue here is not Wavelog or our way to handle different locations. The main issue is that eQSL does not allow the same callsign in different accounts *with overlapping time ranges*. 

There are a few possible ways how to handle that:

###  The way you do it currently 
Always create a new Station Location (using the copy feature) and creating for each location a new *profile* in eqsl. I think this is the way you currently handling that. I see that this is pretty annoying.

### The way I would do it: 
As it's the same callsign you could use just **one main profile** in eQSL. Use a generic picture for this profile and do NOT fill any address information in there. You can leave these fields empty.   

Then you only use two different locations in Wavelog. One for your main QTH and one for your holiday QTH. In this station locations you define the same callsign but different locators and city information. ADDITIONALLY you define a Default QSLMSG with additional information about this QTH.   
<img width="666" alt="Bildschirmfoto 2024-05-22 um 00 57 52" src="https://github.com/wavelog/wavelog/assets/80885850/2588a977-ff88-44d4-bb64-bac7ad5e6eae">

#### The defined QSLMSG gets exported to eQSL and will be printed out on the eQSL card.

<img width="460" alt="Bildschirmfoto 2024-05-22 um 00 41 51" src="https://github.com/wavelog/wavelog/assets/80885850/e912f8a2-d704-4fc0-9a56-b04685429f06">  

You can write whatever you want here. I would write something like "Vy 73 from Vacation QTH in XYZ at [LOCATOR] " or in your main QTH location default qslmsg "Vy 73 from my Home QTH at [LOCATOR] "

This way you can handle two locations with the same callsign in overlapping time periods. The only negative point is, that you only can choose one picture for both... But in my honest opinion this is worth it to avoid such a "location - eqsl profile - mess" like you have. 

------

About your idea/request. I don't see a feature to link more then one eqsl profile (QTH Nickname) to one station location. The amount of backend code for this would not be equal to the use cases out there as your problem is pretty specific in my eyes. It can be solved the way I explained above with a little compromise (just one pricture). Of course, you would also have to change the profile information in eQSL the way so another ham can understand where the QSO was made from. But this is something you can easily do in the WebUI of eQSL (in Profile Settings and QSL Card designer). 

What do you think about that?

**Comment by R1BLH** [comment]
>The amount of backend code for this would not be equal to the use cases out there as your problem is pretty specific in my eyes.

You are right! The game isn't worth the candle.
Thanks for the idea. But the gridsquare is necessary for the eQSL profile. It would be strange to see two gridsquares on the QSL card. :)
So leave it as it is.
